vlib/v/checker/tests/import_symbol_fn_private_err.out

```diff
@@ -0,0 +1,11 @@
+vlib/v/checker/tests/import_symbol_fn_private_err.vv:1:20: error: module `time` function `since()` is private
+    1 | import time { now, since }
+      |                    ~~~~~
+    2 | fn main() {
+    3 |   since(now())
+vlib/v/checker/tests/import_symbol_fn_private_err.vv:3:3: error: function `time.since` is private. curmod=main fmod=time
```

Comment:
I think this message is not clear enough, I think it was used for debugging perhaps?

Reply:
hi @Delta456 I think it was used for debugging, it was preexisting

Reply:
Ok, I believe we could use the same error message for this case to i.e. `error: module `time` function `since()` is private`.

Reply:
Good point the errors should probably be the same, im not sure which one is best.
I feel like `error: module time function since() is private` doesn't flow too well for some reaosn

Reply:
I think this is outside the scope of this PR.

Reply:
How about `error: time module function since() is private` or  `error: time module function's since() is private`?

Reply:
A separate PR will be best for that @Delta456 .

Reply:
See d38bca5 .

Reply:
Nice